Sources/MapboxMaps/Foundation/MapViewMetricsReporter.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+import MetalKit
+
+@_spi(Metrics) public protocol MapViewMetricsReporter {
```

Comment:
```suggestion
@_spi(Metrics) public protocol MapViewMetricsReporter: AnyObject {
```
Performance 🚀

Reply:
I'm not quite following. Why should we require the object to be a class?